Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -51,7 +51,7 @@ PROFILE_GENERATE_OR_USE_CFLAGS = $(PROFILE_GENERATE_OR_USE_CFLAGS_$(ENABLE_LLVM_
 PROFILE_GENERATE_OR_USE_LDFLAGS_ENABLE_LLVM_PROFILE_GENERATION = -fprofile-instr-generate -DENABLE_LLVM_PROFILE_GENERATION=1;
 PROFILE_GENERATE_OR_USE_LDFLAGS = $(PROFILE_GENERATE_OR_USE_LDFLAGS_$(ENABLE_LLVM_PROFILE_GENERATION));
 
-OTHER_CFLAGS = $(inherited) $(PROFILE_GENERATE_OR_USE_CFLAGS);
+OTHER_CFLAGS = $(inherited) -DRELEASE_WITHOUT_OPTIMIZATIONS $(PROFILE_GENERATE_OR_USE_CFLAGS);
```

Comment:
(As discussed privately) let's move this into the per-file compiler flags, so that it's stored alongside the `-O0` that necessitates it.